app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/ide.tsx

```diff
@@ -0,0 +1,74 @@
+/** @file  */
+import * as react from "react";
+
+import * as service from "../service";
+
+const IDE_CDN_URL = "https://ensocdn.s3.us-west-1.amazonaws.com/ide";
+
+interface Props {
```

Comment:
We are trying not to use abbreviations in class and functions names. Abbreviations are ok in local variables in Fns, but lets keep the names full otherwise, please.

Reply:
if you mean `interface Props`, i believe the props interfaces for all other components are also called `Props` - should those all be changed too?